mount.sh

```diff
@@ -1,2 +1,4 @@
+#!/bin/bash
+
 cd src
-python3 main.py /home/sfs
\ No newline at end of file
+python3.8 main.py /fuse_mount
```

Comment:
the mount script will get removed anyway, but besides that not everyone uses 3.8, python should resolve the version by itself

Reply:
That is specifically for my machine. There are multiple conflicting python versions on it.